src/runtime/Makefile

```diff
@@ -239,6 +239,7 @@ ifneq (,$(QEMUCMD))
     # qemu-specific options (all should be suffixed by "_QEMU")
     DEFBLOCKSTORAGEDRIVER_QEMU := virtio-scsi
     DEFNETWORKMODEL_QEMU := tcfilter
+    KERNELTYPE = uncompressed
```

Comment:
@devimc Nice. The numbers look good. I wonder what was the reason we chose to use the compressed kernel for kata (w/ qemu), besides saving spaces on the host (~10M vs ~50M)?

Reply:
this is a new feature introduced in QEMU 5